src/PSRule.Rules.Azure/rules/Azure.RedisEnterprise.Rule.yaml

```
@@ -0,0 +1,53 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+#
+# Rules for Redis Cache Enterprise
+#
+
+#region Rules
+
+
+---
+# Synopsis: Redis Cache Enterprise should reject TLS versions older than 1.2.
+apiVersion: github.com/microsoft/PSRule/v1
+kind: Rule
+metadata:
+  name: Azure.RedisEnterprise.MinTLS
+  ref: AZR-000299
```

@simone-bennett Almost there :) Someone took `AZR-000299` try `AZR-000301`.

```suggestion
  ref: AZR-000301
```